home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1171,6 +1171,98 @@ paths:
         default:
           $ref: '#/components/responses/InternalServerError'
 
+  /polls/{poll_id}/houses:
```

Comment:
As `/polls/{poll_id}/houses` is a resource we have to be consistent in response models among all operations on this resource. 

I think we have to represent houses here. It will be enough.

Please check that all operations here that intended to return content returns the same model.

Reply:
makes sense